From: Sven Roederer <devel-sven@geroedel.de>
Date: Fri, 25 Oct 2019 12:15:30 +0200
Subject: tools/u-boot: enlage max pathlength for mkimage

this prevents build-errors as of "Image file name (mkimage) too long, can't create tmpfile"
On Freifunk-Berlins buildbots such errors have been triggered by depth of directories and
long imagenames.

diff --git a/tools/mkimage/patches/900-mkimage-enlarge-max-pathlen.patch b/tools/mkimage/patches/900-mkimage-enlarge-max-pathlen.patch
new file mode 100644
index 0000000000000000000000000000000000000000..2a76893854e02300f3f466616e450d7041f33807
--- /dev/null
+++ b/tools/mkimage/patches/900-mkimage-enlarge-max-pathlen.patch
@@ -0,0 +1,12 @@
+diff -ur u-boot-2018.03.orig/tools/mkimage.h u-boot-2018.03/tools/mkimage.h
+--- u-boot-2018.03.orig/tools/mkimage.h	2019-10-24 23:57:49.956460361 +0200
++++ u-boot-2018.03/tools/mkimage.h	2019-10-25 12:09:39.002580001 +0200
+@@ -41,7 +41,7 @@
+ }
+ 
+ #define MKIMAGE_TMPFILE_SUFFIX		".tmp"
+-#define MKIMAGE_MAX_TMPFILE_LEN		256
++#define MKIMAGE_MAX_TMPFILE_LEN		384
+ #define MKIMAGE_DEFAULT_DTC_OPTIONS	"-I dts -O dtb -p 500"
+ #define MKIMAGE_MAX_DTC_CMDLINE_LEN	2 * MKIMAGE_MAX_TMPFILE_LEN + 35 /* 35 bytes for quotes, space, -o, len(MKIMAGE_DEFAULT_DTC_OPTIONS), len(MKIMAGE_TMPFILE_SUFFIX) */
+ 
